app/interactors/backdate/destroy_interactor.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Backdate::DestroyInteractor
+  include Interactor
+
+  delegate :params, :edition, :user, to: :context
+
+  def call
+    Edition.transaction do
+      find_and_lock_edition
+      update_edition
+      create_timeline_entry
+    end
+  end
+
+private
+
+  def find_and_lock_edition
+    context.edition = Edition.lock.find_current(document: params[:document])
```

Comment:
I think we should have a check for whether the document is editable here as per: https://github.com/alphagov/content-publisher/blob/9eb34fc4080532947c0d3c3ae6b9cae246cefba1/app/interactors/backdate/update_interactor.rb#L28-L32

Reply:
We're now raising an error if the edition is not editable and a first edition.